borg/archive.py

```diff
@@ -261,6 +262,7 @@ def save(self, name=None, timestamp=None):
             'username': getuser(),
             'time': start.isoformat(),
             'time_end': end.isoformat(),
+            'chunker_params': self.chunker_params,
```

Comment:
i don't think that can break something, but you have to be careful when reading this as it might not be there.